pkg/solana/fees/computebudget_test.go

```diff
@@ -136,6 +136,44 @@ func testSet[V instruction](t *testing.T, builder func(uint) V, setter func(*sol
 		assert.NoError(t, err)
 		assert.Equal(t, data, []byte(tx.Message.Instructions[computeIndex].Data))
 	})
+
+	t.Run("with_lookuptables", func(t *testing.T) {
+		t.Parallel()
+		receiver, err := solana.NewRandomPrivateKey()
+		require.NoError(t, err)
+
+		// build base tx (no fee)
+		tx, err := solana.NewTransaction([]solana.Instruction{
+			system.NewTransferInstruction(
+				0,
+				key.PublicKey(),
+				receiver.PublicKey(),
+			).Build(),
```

Comment:
Since we could have a scenario where we need to adjust multiple indices, could we also test that here? I assume you'd probably have to use a custom instruction here.

Reply:
adjusted the test case to have multiple instructions and multiple indices to adjust!